### PR TITLE
APi-44: added 'is_running' field to get gpw response

### PR DESCRIPTION
**Pull request by van-der-wolf**
I made some reformatting to documentation and removed fields which doesn't exist in actual response from api

### CODE REVIEW

File: source/includes/_gpw.md [hunk 1]
```diff
@@ -283,40 +283,43 @@ var results = request.Get("/v4/gpw/" + reportId + "", parameters);
...
+        "address2": null,
+        "city": "Chicago, IL",
+        "phone_number": null,
+        "search_terms": [
+            "search_term1",
+            "search_term2",
+            "search_term3"
+        ],
+        "google_location": "Chicago, IL",
+        "notify": true,
```
Comment (redhatua) [inline source/includes/_gpw.md]: So "Yes" or just `true`?

File: source/includes/_gpw.md [hunk 2]
```diff
@@ -283,40 +283,43 @@ var results = request.Get("/v4/gpw/" + reportId + "", parameters);
...
+        "email_addresses": [
+        	"email@company.com"
+        ],
+        "notify_about_changes": true,
+        "email_addresses_for_changes": [
+        	"email@company.com"
+        ],
+        "is_public": "Yes",
+        "public_key": "<hidden>",
+        "is_running": false
```
Comment (redhatua) [inline source/includes/_gpw.md]: Or "No"?

File: source/includes/_gpw.md (hunk 1 above)
Comment (ejeliot) [reply source/includes/_gpw.md]: For consistency should be "Yes" and "No" rather than true/false.

**Review by ejeliot — CHANGES_REQUESTED** [review]

**Comment by van-der-wolf** [comment]
@redhatua @ejeliot Fixed

**Review by ejeliot — APPROVED** [review]

**Pull request merged**